src/js/server/models/Hacker.ts

```diff
@@ -204,6 +258,8 @@ const Hacker: Hacker = db.define<HackerInstance, HackerAttributes>('hacker', att
     getApplicationStatus,
     getRsvpStatus,
     getTicketStatus,
+    getStatuses,
+    deriveOverallStatus,
```

Comment:
Is this still being used outside this class?  If not we can remove it from this list

Reply:
The build should fail to compile if the function is missing here but used elsewhere right?

Reply:
Lol evidently the answer is no—I think it is the case with webpack though

Reply:
Okay this has now been refactored